### PR TITLE
Fix issue in query parameters with enums

### PR DESCRIPTION
**Pull request by vithu30**

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.endpoint.registry/swagger.json [hunk 1]
```diff
@@ -323,7 +323,7 @@
             "type" : "string"
           }
         }, {
-          "name" : "sortBy",
+          "name" : "sortByEntry",
```
Comment (fazlan-nazeem) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.endpoint.registry/swagger.json:326]: sortEntryBy?

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.endpoint.registry/swagger.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
             "type" : "string"
           }
         }, {
-          "name" : "sortBy",
+          "name" : "sortByRegistry",
```
Comment (fazlan-nazeem) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.endpoint.registry/swagger.json:31]: sortRegistryBy

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.endpoint.registry/swagger.json (hunk 1 above)
Comment (vithu30) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.endpoint.registry/swagger.json]: Fixed

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.endpoint.registry/swagger.json (hunk 2 above)
Comment (vithu30) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.endpoint.registry/swagger.json]: Fixed

**Pull request merged**